### ISSUE TITLE
closeWindow failed after following the link

### ISSUE THREAD

**Issue by gregatp** [DevExpress/testcafe#5637]
### What is your Test Scenario?

- Open new window
- Search some result
- Click one of result link
- Close the window
 
### What is the Current behavior?

- TestCafe crashes (all windows are closed) with the following error:
> Cannot close the window because it does not have a parent. The parent window was closed or you are attempting to close the root browser window where tests were launched.
  
### What is the Expected behavior?

- Only selected window is closed, no error message is displayed
 
### What is your web application and your TestCafe test code?

 
<details>
<summary>Your website URL (or attach your complete example):</summary>

[http://devexpress.github.io/testcafe](http://devexpress.github.io/testcafe)
</details>

<details>
<summary>Your complete test code (or attach your test files):</summary>

 
```js
import { Selector } from 'testcafe';

fixture `Close browser window`
    .page `http://www.example.com/`;

test('Open new window and close it', async t => {
    await t
        .openWindow('http://devexpress.github.io/testcafe')
        .click(Selector('#search-icon'))
        .wait(1000)
        // new window is closed as expected here
        .closeWindow()
        .openWindow('http://devexpress.github.io/testcafe')
        .click(Selector('#search-icon'))
        .typeText(Selector('#search'), 'closeWindow')
        .click(Selector('a[aria-label="Link to the result"]'))
        .wait(1000)
        // but this time TestCafe crashed wit error:
        // "Cannot close the window because it does not have a parent"
        .closeWindow();
}); 
```
</details>

<details>
<summary>Your complete configuration file (if any):</summary>

```

```
</details>

<details>
<summary>Your complete test report:</summary>

```
 Close browser window
 ✖ Open new window and close it

   1) Cannot close the window because it does not have a parent. The parent window was closed or you are attempting to
      close the root browser window where tests were launched.

      Browser: Chrome 86.0.4240.80 / macOS 10.15.7

         15 |        .typeText(Selector('#search'), 'closeWindow')
         16 |        .click(Selector('a[aria-label="Link to the result"]'))
         17 |        .wait(1000)
         18 |        // but this time TestCafe crashed wit error:
         19 |        // "Cannot close the window because it does not have a parent"
       > 20 |        .closeWindow();
         21 |});
         22 |
```
</details>

<details>
<summary>Screenshots:</summary>

![close-window](https://user-images.githubusercontent.com/56645176/96233288-8114c600-0f98-11eb-9389-3b789ff31a93.png)
</details>

### Steps to Reproduce:

- run the test script attached above
 
### Your Environment details:
 
* testcafe version: 1.9.4                  
* node.js version: v13.14.0                   
* command-line arguments: yarn testcafe chrome tests/closewindow-example.js 
* browser name and version: Chrome 86.0.4240.80 
* platform and version: macOS 10.15.7         
* other:

**Comment by Ogurecher** [comment]
@gregatp, Thank you for your report. I reproduced the issue. We'll update this thread once it's fixed.

For the team:
The same error also appears when using `t.navigateTo` in a child window.
```js
fixture `Close browser window`
    .page `http://www.example.com/`;

test('Open new window and close it', async t => {
    await t
        .openWindow('http://devexpress.github.io/testcafe')
        .navigateTo('http://www.google.com')
        .closeWindow();
}); 
```

**Comment by gregatp** [comment]
This works fine in version v1.14.0
It seems that it has been fixed as side effect of release v1.11.0
